rosbag2/test/rosbag2/test_writer.cpp

```diff
@@ -51,7 +51,7 @@ class WriterTest : public Test
   std::shared_ptr<StrictMock<MockConverterFactory>> converter_factory_;
   std::unique_ptr<MockMetadataIo> metadata_io_;
   std::unique_ptr<rosbag2::Writer> writer_;
-  rosbag2::StorageOptions storage_options_;
+  rosbag2::StorageOptions storage_options_{};
```

Comment:
You probably want this for everyone or no one.

Reply:
I'll move initialization into the constructor